examples/text_classification/ios/TextClassifier/Helper/Define.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+enum Model: String, CaseIterable {
+    case mobileBert = "Mobile Bert"
+    case avgWordClassifier = "Avg Word Classifier"
+
+    var modelPath: String? {
+        switch self {
+        case .mobileBert:
+            return Bundle.main.path(
+                forResource: "bert_classifier", ofType: "tflite")
+        case .avgWordClassifier:
+            return Bundle.main.path(
+                forResource: "average_word_classifier", ofType: "tflite")
+        }
+    }
+}
+
+struct Texts {
+    static let defaultText = "This app consists of a single screen containing a large text field. Below the text field has a button titled classify that, when clicked, runs text classification on the text in the text field. Results as a list where every row item is the label and the confidence in that label."
```

Comment:
Let's not use the design doc description for default text here :) We can switch this to what Duy used for Android: "Google has released 24 versions of the Android operating
        system since 2008 and continues to make substantial investments to develop, grow,
        and improve the OS."